paper/paper.md

```diff
@@ -94,11 +107,27 @@ override fun simulate(node: NodeType): Double {
 }
 ```
 
+![Reversi Heuristic. \label{fig:reversi-heu} ](reversi-heu.png?raw=true "Title")
+
 # Results
 
 When the MCTS solver is accurately selecting the optimal solutions, it will continue to compel the agent to explore in the optimal subset of the state space, and reduce its exploration in the non-optimal subset of the state space. We provide a quick example in the MDP Domain of GridWorld, detailed in [@Liu:2021]. The cumulative number of visits corresponding to the optimal policy is proportionally increasing with respect to the number of MCTS iterations. Whereas for non-optimal solutions, the cumulative visits are significantly lower because the solver will avoid visiting the non-optimal state subspace. 
 
-![Convergence of visits](gw_visits.png?raw=true "Title")
+![Convergence of visits.](gw_visits.png?raw=true "Title")
+
+# Real World Applications
+
+*mctreesearch4j* is designed to run on the JVM, and therefore is useful in a research setting where JVM is intended to be used alongside either the implementation of the game or the core AI engine. Typically, for state-of-the-art AI, MCTS is used in combination with other Deep Learning algorithms, where MCTS drives the exploration of hypothetical games to be played out in order for the agent to make better decisions.
+
+MCTS is typically combined with Deep Learning strategies to achieve state-of-the-art performance for complex games whose solutions are typically intractable via standard tree-search methods. For example in AlphaGo [silver:2016], MCTS is combined with two types of Neural Networks, namely Value Networks and Policy Networks, to achieve state-of-the-art performance in the game of Go, never achieved before. Similarly, in the area of control theory, MCTS acting in combination with Value and Policy Networks to produce state and action space distributions, have shown impressive performance in probabilistic planning for self-driving cars, as demonstrated in Tesla’s AI day event [tesla:2021]. The utility of MCTS illustrates the acceptance of MCTS into the standard repetoire of both probablisitc and deterministic planning. And as the need for MCTS grows among industry and research, the impetus to create MCTS software libraries that are standardized, reliable, and can integrate well into general software ecosystems, such as JVM, is increasing.
```

Comment:
"Neural Networks", "Policy Networks", etc. Seeing some random capitalization here.

Reply:
Correction made in https://github.com/aqtech-ca/mctreesearch4j/pull/70